packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -1945,7 +1947,31 @@ fn run_experiment(
             v8::V8::initialize_platform(platform);
             v8::V8::initialize();
 
-            let mut isolate = v8::Isolate::new(Default::default());
+            let mut create_params = v8::Isolate::create_params();
+
+            create_params = match (
+                init_msg.js_runner_initial_heap_constraint,
+                init_msg.js_runner_max_heap_size,
+            ) {
+                (None, None) => create_params,
+                (None, Some(js_runner_max_heap_size)) => {
+                    create_params.heap_limits(0, js_runner_max_heap_size * MB)
```

Comment:
Can we add comments to these explaining that 0 is rusty_v8 (or V8?)'s defaults

Reply:
Added comment in https://github.com/hashintel/hash/pull/446/commits/e8ff292852c45d60fe6560c2e6757a112cac209a